civis/io/_tables.py

```diff
@@ -393,7 +382,7 @@ def civis_to_csv(filename, sql, database, job_name=None, api_key=None,
 
     # always set include_header to False and compression to gzip to
```

Comment:
Should this comment about setting include_header to False change now?

Reply:
updated comment; we're no longer setting include_header and just passing through what the user provided